<?xml version="1.0"?>
<launch>
    <arg name="flag" default="linear" />
    <include file="$(find franka_example_controllers)/launch/hiro_joint_velocity_example_controller.launch" pass_all_args="true"/>
    <include file="$(find relaxed_ik_ros1)/launch/demo.launch" pass_all_args="true"/>
    <node pkg="relaxed_ik_ros1" type="xbox_input_test.py" name="xbox_input" output="screen">
        <param name="flag" value="$(arg flag)" />
    </node>
    <node pkg="relaxed_ik_ros1" type="gui_option1.py" name="gui_option1" output="screen">
        <param name="config_file" value="$(find relaxed_ik_ros1)/scripts/Hades.json"/>
    </node>
    <node pkg="joy" type="joy_node" name="joy" output="screen" />
    <!-- <node pkg="relaxed_ik_ros1" type="SelfCapSkinPatch2x2Plot.py" name="skin_touch" output="screen" /> -->
</launch>